packages/ListBox/src/components/Options/helpers/options.js

```diff
@@ -0,0 +1,234 @@
+import useListBox from "../../../useListBox";
```

Comment:
I think I prefer the way you've organized the `Filter` folder — it's flat.  But maybe it would be annoying to have a bunch of `helpers.js` files in your editor 🤔

What about flat, but prefixed?  `filterHelpers.js`, `optionsHelpers.js`, etc?

Reply:
agree is part of the road map for improving this component.